src/components/menu-item/menu-item.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, fixture, html, waitUntil } from '@open-wc/testing';
+import sinon from 'sinon';
+import type SlMenuItem from './menu-item';
+
+describe('<sl-menu-item>', () => {
+  it('passes accessibility test', async () => {
+    const el = await fixture<SlMenuItem>(html`
+      <sl-select>
+        <sl-menu-item>Test</sl-menu-item>
+      </sl-select>
+    `);
+    await expect(el).to.be.accessible();
+  });
+
+  it('default properties', async () => {
+    const el = await fixture<SlMenuItem>(html` <sl-menu-item>Test</sl-menu-item> `);
+
+    expect(el.checked).to.be.false;
+    expect(el.getAttribute('aria-checked')).to.equal('false');
+    expect(el.value).to.equal('');
+    expect(el.disabled).to.be.false;
+    expect(el.getAttribute('aria-disabled')).to.equal('false');
+  });
+
+  it('changes aria attributes', async () => {
+    const el = await fixture<SlMenuItem>(html` <sl-menu-item>Test</sl-menu-item> `);
+
+    el.checked = true;
+    setTimeout(() => expect(el.getAttribute('aria-checked')).to.equal('true'));
```

Comment:
To keep things consistent, can we use `aTimeout()` from `@open-wc/testing` for these?

Reply:
I changed it and I see that you already merged it. I noticed that other tests also use `setTimeout` instead of `aTimeout`. Should I also open a PR to change them?